public/robots.txt

```diff
@@ -1,5 +1,5 @@
 # robots.txt for measured.co
 
 User-agent: *
-Disallow /edit/
+Disallow: */edit
```

Comment:
Hm I'm not sure we need this rule at all because `/edit` will 404 if the user is not logged in

Reply:
Not that it hurts.

Reply:
Even better! Let me fix that up quickly.

Reply:
OK, just killed the rule so we don't have a syntax issue.